packages/site/src/data/svelte/blogs.ts

```diff
@@ -85,16 +85,6 @@ export const blogs: Blog<(typeof blogTags)[number]>[] = [
 		href: 'https://magrippis.com/blog',
 		tags: [],
 	},
-	{
-		title: 'Rodney Lab',
-		author: 'Rodney',
-		description:
-			'Hello and welcome! I’m Rodney, a web developer based in the UK 🇬🇧 and am available for consultancy work.',
-		image:
-			'https://rodneylab.com/assets/rodney-johnson-about-rodneylab-1350x1350.208d1490.avif',
-		href: 'https://rodneylab.com/tags/sveltekit/',
-		tags: [],
-	},
```

Comment:
Curious, why was this one removed? At first glance, it does look like a bunch of SvelteKit blogs.

Reply:
This was a duplicate.

So I had to remove it.